test/api/user/get-installed-packages.test.js

```diff
@@ -106,7 +106,7 @@ test.before(async (t) => {
 
     for (const pkg of testPkgs(t.context.userId1)) {
       const { id } = await db.package.create(pkg)
-      await db.package.updatePackageInstallsInternal({ id, installs: pkg.installs })
+      await db.package.updatePackageInternalDANGER({ id, installs: pkg.installs })
```

Comment:
rather than having a generic danger func, i think this is a case where a non-dangerous db func that adds installs to a package can be added to the class and used here. we already have the logic in session-complete. having it in two places definitely smells bad but i think it smells worse to have this thing

Reply:
im just going to update the package in the test, we shouldn't have a method that sets installs or revenue fields